test/application.factory.ts

```diff
@@ -0,0 +1,45 @@
+import * as bodyParser from 'body-parser'
+import { Application } from 'express'
+import { Container } from 'inversify'
+import { buildProviderModule } from 'inversify-binding-decorators'
+import { InversifyExpressServer } from 'inversify-express-utils'
+import { Environment } from 'nunjucks'
+import * as nunjucks from 'nunjucks'
+
+import { asGovUKErrorList } from 'app/filters/asGovUKErrorList.filter'
+
+function addFilters(env: Environment): void {
```

Comment:
I think this function should be placed somewhere in src/ so that it can be resued here and in NunjucksLoader